python/Ganga/test/GPI/Internals/TestRepo.py

```diff
@@ -16,7 +16,10 @@ def __init__(self, name):
     def _read_access(self, root, obj):
         self.repo.load([root._registry_id])
 
-    def _write_access(self, root):
+    def _acquire_session_lock(self, root):
```

Comment:
Is it better to lock the methods here in the same way as they're locked in the actual Registry?
I'm weary that this test could be hiding possible deadlocks if it doesn't reflect the locking in the actual registry